src/EnergyPlus/SetPointManager.hh

```diff
@@ -61,7 +61,10 @@ namespace EnergyPlus {
 
 namespace SetPointManager {
 
-    // Using/Aliasing
+    enum class CtrlNodeType: int {
+        control,
+        reference
+    };
```

Comment:
I'm not up to speed on these enums so critique away

Reply:
👍 👍 the int type specifier isn't necessarily, but perfectly fine.